admin_guide/garbage_collection.adoc

```diff
@@ -96,22 +96,22 @@ endif::[]
 
 Each spin of the garbage collector loop goes through the following steps:
 
-1. Retrieve a list of available containers.
-2. Filter out all containers that are running or are not alive longer than
-the `*minimum-container-ttl-duration*` parameter.
-3. Classify all remaining containers into equivalence classes based on pod and image name membership.
-4. Remove all unidentified containers (containers that are managed by kubelet but their name is malformed).
+1. Retrieves a list of available containers.
+2. Filters out all containers that are running or are not alive longer than
+the `*minimum-container-ttl-duration*` parameter. Docker containers that are not alive can be either in an exited or dead state. Kubernetes containers that are not alive are in a terminated state.
```

Comment:
We really want to avoid any references to Docker in our doc set. I don't think we need to make a distinction between container type here, but you can list the example states. It's a tricky one! I discussed this with @adellape a bit and he located https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/, which is a nice resource. Part of the issue in the BZ is that this whole topic needs to be consistent in usage of "container states" vs. "pod states" . We might need some more SME input here.

Reply:
Agree! We could just say containers instead of Docker containers here.

Reply:
Have changed this to just "containers".